docs/user-guide/global-configurations/sso-login.md

```diff
@@ -26,17 +26,15 @@ For examples see
 
 Login as a user with super-admin privileges and go to `Global Configurations` -&gt; `SSO Login Services` and click on any `Identity Provider` and fill the configuration. 
 
-Add valid devtron application `URL` where it hosted.
+Add valid devtron application `URL` where it is hosted.
 
-Fill correct `redirect URL` or `callback URL` with which you have registered with the identity provider in the previous step along with the `client id` and `client secret` shared by the identity provider.
+Fill correct `redirect URL` or `callback URL` from which you have registered with the identity provider in the previous step along with the `client id` and `client secret` shared by the identity provider.
 
-Only single SSO login configuration can be active at one time. Whenever you create or update any sso config that config will be activated and used by the system and previous configuration will be deleted.
-
-![](../../.gitbook/assets/sso-login-3.png)
+Only single SSO login configuration can be active at one time. Whenever you create or update any SSO config, it will be activated and used by the system and previous configurations will be deleted.
 
 URL and redirectURI should be same as provided in screenshots except domain substring.
 
-![](../../.gitbook/assets/sso-login-4.png)
+![](../../.gitbook/assets/sso-config)
```

Comment:
Why does image file has no extension?

Reply:
Not required in this case, it will work.

Reply:
Please add the jpg format as it helps in SEO purpose

Reply:
Added, the jpg extension.

Reply:
You missed to update the link here

Reply:
updated.